fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/AggregateIndexMatchCandidate.java

```diff
@@ -202,9 +217,12 @@ public List<MatchedOrderingPart> computeMatchedOrderingParts(@Nonnull final Matc
             //
             // Compute a Value for this normalized key.
             //
+            final var value = deconstructedValue.get(permutedIndex).rebase(aliasMap);
+            /*
```

Comment:
please take this comment out -- and maybe add a comment

Reply:
Done